core/lib/tasks/core.rake

```
@@ -192,6 +192,49 @@ use rake db:load_file[/absolute/path/to/sample/filename.rb]}
+        say "Migrated #{policy_slug} to store #{store.id}"
+      end
+    end
+
+    Spree::PageLink.where(linkable_type: 'ActionText::RichText').each do |page_link|
+      # Check if this page link is attached to ActionText content
+      # Find the ActionText record
+      action_text = ActionText::RichText.find_by(id: page_link.linkable_id)
+      next unless action_text
+
+      # Check if the ActionText belongs to a store policy
+      if action_text.record_type == 'Spree::Store' && action_text.name.to_s.include?('policy')
```

RuboCop (1.76.1)

[convention] Use `next` to skip iteration.

(Style/Next)